examples/va/hci/README.md

```diff
@@ -0,0 +1,39 @@
+# Hyperconverged OpenStack and Ceph
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+This is a collection of CR templates that represent a validated Red Hat OpenStack Services on OpenShift deployment that has the following characteristics:
+
+- 3 master/worker combo-node OpenShift cluster
+- 3-replica Galera database
+- RabbitMQ
+- OVN networking
+- Network isolation over a single NIC
+- 3 compute nodes
+- CephHCI installed on compute nodes and used by various OSP services
+    - Cinder Volume using RBD for backend
+    - Cinder Backup using RBD for backend
+    - Glance using RBD for backend
+    - Nova using RBD for ephemeral storage
+    - Manila using CephFS for backend
+
+## Considerations
+
+1. These CRs are validated for the overall functionality of the OSP cloud deployed, but they nonetheless require customization for the particular environment in which they are utilized.  In this sense they are _templates_ meant to be consumed and tweaked to fit the specific constraints of the hardware available.
+
+2. The CRs are applied against an OpenShift cluster in _stages_.  That is, there is an ordering in which each grouping of CRs is fed to the cluster.  It is _not_ a case of simply taking all CRs from all stages and applying them all at once.
+
+3. In stages 1 and 2 [kustomize](https://kustomize.io/) is used to genereate the CRs dynamically. The `values.yaml` file(s) must be updated to fit your environment. kustomize version 5 or newer required.
+
+4. In stages 3 and 4 YAML comments are placed throughout the CRs to aid in the process of customizing the CRs.  Fields that _must_ (or most likely need to be) changed are commented with "# CHANGEME" either on the field itself or somewhere nearby.  Other comments are added to explain fields that can be changed and, sometimes, to explain additions that can be made.
```

Comment:
Should we remove the mention of `# CHANGEME` here, since we're using `kustomize` for all of that now?  Or are there still places where we rely on `# CHANGEME` for particular reasons?

Reply:
Stages 3 and 4 still have `#CHANGEME` because they have not yet been moved to kustomize. They refer to `edpm-pre-ceph` and `edpm-post-ceph`.

I was thinking of moving them to kustomize in a follow up PR since this change is pretty big.